### ISSUE TITLE
default value of auto_bootstrap in scylla and scylla-ami is inconsistent

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla#2017]
*Installation details*
Scylla version (or git commit hash): latest scylla /scylla-ami
OS (RHEL/CentOS/Ubuntu/AWS AMI): 

`auto_bootstrap` is enabled by default scylla code
```
    val(auto_bootstrap, bool, true, Used,     \
            "This setting has been removed from default configuration. It makes new (non-seed) nodes automatically migrate the right data to themselves. When initializing a fresh cluster with no data, add auto_bootstrap: false.\n"  \
            "Related information: Initializing a multiple node cluster (single data center) and Initializing a multiple node cluster (multiple data centers)."
```

auto_bootstrap is disabled by default in scylla-ami
```
    # Option that enforces a bootstrapping node
    parser.add_argument("--bootstrap", action="store_true", dest="bootstrap", default=False)
```

We can easy configure bootstrap parameter in user-data in creating cluster by AMI. But the problem is RPM installed scylla has different default value with AMI installed instance.

How do we suggest our users about bootstrap when they create a cluster by AMI? We should enable it for db nodes?

CC @nirmaayan

**Comment by tzach** [comment]
This actually makes sense:
- If an AMI is launched as part of a new cluster, there is no data to stream
- If an AMI is launched as a standalone, there is no nodes to stream from

Streaming (auto_bootstrap=true) make sense only if we provide seeds as part of the user data.

**Comment by avikivity** [comment]
Fixes in scylladb/scylla-ami@2319a532f6282967cc9cf7d1d24524e302b86ac0 (streaming defaults to True).
